livekit-agents/livekit/agents/llm/chat_context.py

```diff
@@ -27,22 +27,24 @@ class ChatRole(enum.Enum):
     TOOL = "tool"
 
 
+@dataclass
+class ChatImage:
+    image: str | rtc.VideoFrame
+    inference_width: int = 128
+    inference_height: int = 128
```

Comment:
Motivation for changing to None is to allow to detect "unspecified". So we can chose strong default depending on which LLM is being used. For example Gemini doesn't need any specific size 
```suggestion
    inference_width: int | None = None
    inference_height: int | None = None
```

Reply:
Interesting, yeah I was mostly thinking about a sane default in terms of sending images over the wire (and not thinking as much about the URL case). No strong opinion here, None is fine.